src/utils/polygon.cpp

```diff
@@ -190,6 +190,22 @@ unsigned int Polygons::findInside(Point p, bool border_result)
     return ret;
 }
 
+int64_t Polygons::polyLineLength() const
+{
+    int64_t length = 0;
```

Comment:
Oh a long long is at least 64 bit, so I'm changing this to coord_t.
I'm not going to introduce an unsigned_coord_t because of this comment.....